### PR TITLE
Ispn 5449 java.lang.ClassCastException: [B cannot be cast to...

### PR DESCRIPTION
**Pull request by wburns**
https://issues.jboss.org/browse/ISPN-5449
https://issues.jboss.org/browse/ISPN-4133

This will need to be backported to 7.2 as well.

### CODE REVIEW

File: core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java [hunk 1]
```diff
@@ -313,9 +314,9 @@ public void testRemoteGetArrivesButWriteOccursBeforeRegistration() throws Throwa
       // interceptor
       CyclicBarrier getBarrier = new CyclicBarrier(3);
       ownerCache.getAdvancedCache().addInterceptorAfter(new BlockingInterceptor(getBarrier, GetCacheEntryCommand.class, true, false),
-                                                        getL1InterceptorClass());
+              getL1InterceptorClass());
       backupOwnerCache.getAdvancedCache().addInterceptorAfter(new BlockingInterceptor(getBarrier, GetCacheEntryCommand.class, true, false),
-                                                       getL1InterceptorClass());
+              getL1InterceptorClass());
```
Comment (danberindei) [inline core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java]: Indented at 8 chars instead of 6?

**Comment by danberindei** [comment]
1st commit message: JIRA subject should be on the first line.

File: core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java (hunk 1 above)
Comment (wburns) [inline core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java]: I just let IntelliJ do it's thing :P

**Comment by wburns** [comment]
It actually was on the first line, but git auto wrapping moved it down, haha - I will need to add a space somewhere so it doesn't wrap.

File: core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java (hunk 1 above)
Comment (wburns) [inline core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java]: Yeah the code style import for IntelliJ has continuation indent at 8.  Should we change this?

**Comment by wburns** [comment]
I fixed the commit message and just undid the whitespace changes.

**Comment by danberindei** [comment]
Integrated in master and 7.2.x, thanks Will!

**Pull request closed without merging**